ci/test/03_test_script.sh

```diff
@@ -146,21 +146,74 @@ if [ "$RUN_CHECK_DEPS" = "true" ]; then
   "${BASE_ROOT_DIR}/contrib/devtools/check-deps.sh" .
 fi
 
+function get_interfaces()
+{
+  set -o pipefail
+  ifconfig | awk -F ':| ' '/^[^[:space:]]/ { if (!match($1, /^lo/)) { print $1 } }'
+  set +o pipefail
+}
+
+function tcpdump_file()
+{
+  echo "/tmp/tcpdump_$1_$2"
+}
+
+function traffic_monitor_begin()
+{
+  test_name="$1"
+  for ifname in $(get_interfaces) ; do
+    tcpdump -nU -i "$ifname" -w "$(tcpdump_file "$test_name" "$ifname")" &
+  done
+}
+
+function traffic_monitor_end()
+{
+  test_name="$1"
+
+  for ifname in $(get_interfaces) ; do
+    f=$(tcpdump_file "$test_name" "$ifname")
+    if [ ! -e "$f" ] && [ "$FILE_ENV" != "./ci/test/00_setup_env_native_asan.sh" ] ; then
```

Comment:
nit: someone renaming this file down the line might not be aware of a dependency on this exact file name

Reply:
Yeah, I was thinking the same... Should I change it to the following?

```diff
-     if [ ! -e "$f" ] && [ "$FILE_ENV" != "./ci/test/00_setup_env_native_asan.sh" ] ; then
+     if [ ! -e "$f" ] && [ "$CONTAINER_NAME" != "ci_native_asan" ] ; then
```

Reply:
Yet another alternative would be to set a flag like `CI_FAIL_IF_NO_TCPDUMP_FILE` (or similar) in  00_setup_env_native_asan.sh and add a comment mentioning the dependency:
- if the script name or the container name are renamed, this will still work
- if script is removed, someone might see the comment about the dependency during review
- it's easy to add more tasks to the list of required tasks by just adding `CI_FAIL_IF_NO_TCPDUMP="true"`

Reply:
Done! Much better.